src/components/cards/wallet/WalletHint.tsx

```diff
@@ -0,0 +1,23 @@
+import { Wallet } from "@/types/wallet";
+import Currency from "@/components/ui/Currency";
+import Hint from "@/components/ui/Hint";
+import { useTranslation } from "next-i18next";
+
+interface HintProps {
```

Comment:
I think we use type for components props

Reply:
We are using interface in all components, I think we can stick to same approach